### DISCUSSION TITLE
Problema de Input

### DISCUSSION THREAD

**Discussion by Monti7475** [IIC2133-PUC/2024-1#115] [Tarea 1] [answered]
Vi la discussion que habla acerca de ponerle un \n a los scans, yo tenia ese error y lo hice para poder solucionarlo
Pero ahora me da otro error respecto a los scans
![image](https://github.com/IIC2133-PUC/2024-1/assets/130416325/3cd9677f-a7c9-4611-8c99-eb86887ae867)
Lo raro es, esto me pasa al ejecutarlo "normal" y con valgrind, pero al ejecutarlo con gdb no pasa. Cual podría ser el problema? (La única edición que tiene el código es añadir los \n)

**Comment by benjavicente** [comment]
Hola! Sube ese cambio a GitHub para ver que está pasando 🤔

**Reply by Monti7475** [reply]
Asi como en un push en mi repo? o subo lo que cambie como texto acá

**Reply by Monti7475** [reply, marked as answer]
Ya vi mi problema, no era algo del código
Tenía algo mal en la consola 😓
